### PR TITLE
Manhwa latino: Fix to be able a bypass through Cloudflare

### PR DESCRIPTION
**Pull request by lberoiza**
Possible Fix to go through Cloudflare .

Checklist:

- [OK] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [OK] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [OK] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [OK] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [OK] Have not changed source names
- [OK] Have explicitly kept the `id` if a source's name or language were changed
- [OK] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoParseHttpSource.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+abstract class ManhwaLatinoParseHttpSource : ParsedHttpSource() {
+
+    // Randomaze the userAgent to try to help with cloudflare
+    protected open val userAgentRandomizer = " ${Random.nextInt().absoluteValue}"
+
+    /**
+     * User Agent for this Website
+     */
+    private val userAgent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:77.0) Gecko/20100101 Firefox/78.0$userAgentRandomizer"
```
Comment (Riztard) [inline src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoParseHttpSource.kt:19]: why need a separate file?
cant these be added inside ManhwaLatino.kt?

Comment (lberoiza) [reply src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoParseHttpSource.kt:19]: This file has only the connection functionality, the another one the parse functionality.

File: src/es/mahnwalatino/build.gradle [hunk 2]
```diff
@@ -5,8 +5,8 @@ ext {
     extName = 'Manhwa-Latino'
     pkgNameSuffix = 'es.manhwalatino'
     extClass = '.ManhwaLatino'
-    extVersionCode = 11
-    containsNsfw = true
+    extVersionCode = 14
```
Comment (Riztard) [inline src/es/mahnwalatino/build.gradle:8]: 12? 
after 11 is 12 btw just in case you dont know :v

Comment (CarlosEsco) [reply src/es/mahnwalatino/build.gradle:8]: @lberoiza fix this please

**Comment by lberoiza** [comment]
i am still working on cloudflare issue, the modification was well, for a cupple of Days, but they update the website again.

**Pull request closed without merging**